packages/botonic-dx/sample-config/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "node_modules/botonic-dx/baseline/tsconfig.json",
```

Comment:
Same here, should this be `@botonic/dx/...`?

Reply:
fixed